tests/test_ingest.py

```diff
@@ -0,0 +1,34 @@
+import pathlib
+
+from workflow_ephys.ingest import (ingest_subjects, ingest_sessions)
+from workflow_ephys.pipeline import (subject, lab, ephys, probe, Session,
```

Comment:
Similarly here. These imports will create schemas and tables that are left 'lingering' w/o properly being torndown. See [here](https://github.com/datajoint/pharus/blob/8afc4d1707477ac3325b1da11e041ee3d0666e0d/tests/test_schemas.py#L1) for example. The actual imports all come from the top level directory module which will properly setup and teardown on any schema/table involved.